librecad/src/lib/gui/rs_painter.h

```diff
@@ -50,9 +50,8 @@ class QBrush;
  */
 class RS_Painter {
 public:
-    RS_Painter() {
+    RS_Painter(): offset(RS_Vector(0.0,0.0)) {
```

Comment:
try to use aggregate initialization, even though for most time, it doesn't matter (the case here).

interesting difference:

```C++
struct A;
A a(); // declare a function A a() -> A {...}
A b{}; // define a variable a by default ctor of A
vector<int> v(2, 1); // v contains {1, 1}
vector<int> w{2, 1}; // w contains {2, 1}
```